tests/foreman/virtwho/utils.py

```diff
@@ -303,3 +305,54 @@ def deploy_configure_by_script(script_content, debug=False):
         )
     if debug:
         return deploy_validation()
+
+
+def restart_virt_who_service():
+    """
+    Do the following:
+    1. clean rhsm.log message, make sure there is no old message exist.
+    2. restart virt-who service via systemctl command
+    """
+    runcmd("rm -f /var/log/rhsm/rhsm.log")
+    runcmd("systemctl restart virt-who")
+
+
+def create_etc_d_file(form_data,
+                      rhsm_username='admin',
+                      rhsm_password='admin',
+                      filename='virt-who-config.conf'):
+    filepath = '/etc/virt-who.d/{}'.format(filename)
+    type = form_data['hypervisor_type']
```

Comment:
`type` is built-in function, better to use `type_` or `hypervisor_type`.

Reply:
Forgot this, thanks